src/datagrid/DataGrid.tsx

```diff
@@ -1092,7 +1094,7 @@ export class DataGrid extends React.PureComponent<DataGridProps, DataGridState>
     // Function to create placeholder for empty datagrid
     private buildEmptyPlaceholder(): React.ReactElement {
         const {itemText} = this.state;
-        const placeholderText = "We couldn't find any " + itemText + " !";
+        const placeholderText = "No " + itemText + " found!";
```

Comment:
is this change requested by someone ? We developed this message as per VMware clarity